infra/base-images/base-runner/test_one.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+"""Does bad_build_check on a fuzz target in $OUT."""
+import os
+import sys
+
+import test_all
+
+
+def test_one(fuzz_target):
+  """Does bad_build_check on one fuzz target. Returns True on success."""
+  with test_all.use_different_out_dir():
+    fuzz_target_path = os.path.join(os.environ['OUT'], fuzz_target)
+    return test_all.do_bad_build_check(fuzz_target_path).returncode == 0
```

Comment:
I think to make it easier to debug issues like https://github.com/google/oss-fuzz/issues/5349 it would be great if `test_one` didn't hide the output of `bad_build_check`. I'm not sure but something like
```diff
diff --git a/infra/base-images/base-runner/test_one.py b/infra/base-images/base-runner/test_one.py
index 9bdb75fa..73455f8e 100755
--- a/infra/base-images/base-runner/test_one.py
+++ b/infra/base-images/base-runner/test_one.py
@@ -25,7 +25,12 @@ def test_one(fuzz_target):
   """Does bad_build_check on one fuzz target. Returns True on success."""
   with test_all.use_different_out_dir():
     fuzz_target_path = os.path.join(os.environ['OUT'], fuzz_target)
-    return test_all.do_bad_build_check(fuzz_target_path).returncode == 0
+    result = test_all.do_bad_build_check(fuzz_target_path)
+    if result.returncode != 0:
+      sys.stdout.buffer.write(result.stdout + result.stderr + b'\n')
+      return False
+    return True
```
should probably suffice.